recipes-samples/images/rpb-qt5-image.bb

```diff
@@ -0,0 +1,33 @@
+SUMMARY = "Basic QT5 EGLFS image"
+
+IMAGE_FEATURES += " package-management debug-tweaks ssh-server-openssh hwcodecs tools-debug"
+
+LICENSE = "MIT"
+
+inherit core-image distro_features_check extrausers
+
+# let's make sure we have a good image..
+REQUIRED_DISTRO_FEATURES = "pam systemd opengl"
+
+# make sure we boot to desktop
+# by default and without x11-base in IMAGE_FEATURES we default to multi-user.target
+SYSTEMD_DEFAULT_TARGET = "multi-user.target"
+
+CORE_IMAGE_BASE_INSTALL += " \
```

Comment:
we probably want to keep the x11 packages as well, but check for X11 in DISTRO_FEATURES. I believe this image should be used for DISTRO=rpb and DISTRO=rpb-wayland, and in case of rpb it should be using Qt5 over x11 backend, so the x11 packages would be needed.

in fact we might eventually end up with a single desktop image, who knows..